@xen-orchestra/lite/src/App/TreeView.tsx

```diff
@@ -41,6 +46,17 @@ const getIconColor = (obj: Host | Vm) => {
 
 const TreeView = withState<State, Props, Effects, Computed, ParentState, ParentEffects>(
   {
+    initialState: ({ location }) => ({
+      _selectedNodes: [location.pathname.split('/')[3]],
```

Comment:
Check if `location.pathname.split('/')[3]` is `undefined` to avoid putting it in the array.